app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,68 @@
 @import "bootstrap";
 
+$orange: #f2580b;
+@mixin orange_background {
+  background-color: $orange;
+  border-color: $orange;
+}
+body {
+  background-color: #eee;
+  font-family: 'Roboto', sans-serif;
+}
+
+.navbar-header {
+  border-bottom: 1px solid gray;
+}
+
 .container {
   margin: 48px auto;
   width: 960px;
-}
 
-.container >  h1 {
-  float: left;
-  margin: 0;
+  h1 {
+    float: left;
+    margin: 0;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3